mlflow/tracking/client.py

```diff
@@ -1732,6 +1732,10 @@ def log_artifact(self, run_id, local_path, artifact_path=None) -> None:
             is_dir: False
 
         """
+        if run_id.startswith("tr-"):
```

Comment:
Could you explain what `tr-` prefix means ?

Reply:
Shouldn't we check if the run id exists in current experiment ?

Reply:
It's the same check we do on the backend, but for this `log_artifact` there is no rest API matching it, so we are bringing the check to client. `tr-` is the identifier for trace id.